libnetwork/README.md

```diff
@@ -11,79 +11,6 @@ Please refer to the [design](docs/design.md) for more information.
 
 There are many networking solutions available to suit a broad range of use-cases. libnetwork uses a driver / plugin model to support all of these solutions while abstracting the complexity of the driver implementations by exposing a simple and consistent Network Model to users.
 
-
-```go
-package main
```

Comment:
Do you think the example itself is good to have in godoc? If we think there's still a use for it, we could consider an _actual_ "godoc" example, similar to https://github.com/moby/moby/blob/199793350807908c04b56741b11e6b943158ac51/api/types/filters/example_test.go#L4-L35

Which shows in the docs like https://pkg.go.dev/github.com/docker/docker/api/types/filters#example-Args.MatchKVList

Reply:
Although libnetwork isn't an internal package, we don't expect any other project to use it. So I think it's not worth re-introducing this snippet as a doc example (although I agree we should fix the pkg description).

Reply:
Yeah, makes sense. Thought I'd mention it as an option before we rip it out, but agreed that it's probably not worth the effort to write new examples.